core/src/test/kotlin/InviteTests.kt

```diff
@@ -0,0 +1,24 @@
+import com.gitlab.kordlib.cache.api.put
+import com.gitlab.kordlib.core.Kord
+import com.gitlab.kordlib.core.supplier.EntitySupplyStrategy
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import kotlin.test.assertEquals
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class InviteTests {
+
+    @Test
+    fun `get invite from rest and cache`() = runBlocking {
+        val kord = Kord(System.getenv("KORD_TEST_TOKEN"))
+        val invite = kord.defaultSupplier.getInvite("mpDQm5N")
```

Comment:
Should probably store the invite code in an ENV variable instead. Don't want to update our source code because our invite expired or whatever.